packages/ai-constructs/API.md

```diff
@@ -63,6 +63,12 @@ type ConversationTurnEvent = {
         modelId: string;
         systemPrompt: string;
         region?: string;
+        inferenceConfiguration?: {
+            maxTokens?: number;
+            temperature?: number;
+            topP?: number;
+            stopSequences?: string[];
```

Comment:
We don't need to define `stopSequences` currently for the conversation route, so the event payload coming from AppSync won't include it.

I think the answer is yes, but are we confident that the lambda runtime code can gracefully handle custom `stopSequences`?
If yes, I'm not opposed to keeping it -- if we ever added support in schema builder + appsync resolver layer, we wouldn't need to update the lamba code.

If not, maybe we don't include it for now.